lib/bolt/cli.rb

```diff
@@ -107,6 +149,16 @@ def parse_command
         # help text
         options[:subcommand] = nil unless COMMANDS.include?(options[:subcommand])
 
+        if Bolt::Util.first_run? && Bolt::Config.user_path
```

Comment:
```suggestion
        if Bolt::Util.first_run?
```

This will still fail if the homedir doesn't exist, since `first_run?` will call `first_runs_free` and try to concatenate with `nil`.